src/components/EntityInfo.vue

```diff
@@ -33,88 +34,96 @@
+            <span v-if="downloadSummaryEnabled">
+              <v-btn
+                small text color="primary"
+                id="download-summary-button"
+                @click="downloadBusinessSummary()"
+              >
+                <v-icon medium>mdi-file-pdf-outline</v-icon>
+                <span>Download Summary</span>
+              </v-btn>
+            </span>
+          </menu>
+        </v-col>
+
+        <v-col cols="12" md="3">
+          <dl>
+            <!-- Business Number -->
+            <template v-if="businessId">
+              <dt class="mr-2">Business Number:</dt>
+              <dd id="entity-business-number">{{ entityBusinessNo || 'Not Available' }}</dd>
+            </template>
 
-              <v-menu offset-y>
-                <template v-slot:activator="{ on }">
-                  <!-- FUTURE? -> remove disabled prop -->
-                  <v-btn id="menu-down-button" small icon color="primary" v-on="on" disabled>
-                    <v-icon small>mdi-menu-down</v-icon>
-                  </v-btn>
-                </template>
-
-                <v-list class="pt-0 pb-0">
-                  <v-list-item id="business-summary-menuitem" @click="downloadBusinessSummary()">
-                    <v-icon>mdi-download</v-icon>
-                    <v-list-item-title class="pl-1">Download Business Summary</v-list-item-title>
-                  </v-list-item>
-                </v-list>
-              </v-menu>
-            </div>
-
-            <!-- Business Email, Phone -->
-            <dl class="profile-info">
-              <dt><span class="sr-only mr-2">Business Email:</span></dt>
-              <dd id="entity-business-email">{{businessEmail || 'Unknown Email'}}</dd>
-              <template v-if="fullPhoneNumber">
-                <dt><span class="sr-only mr-2">Business Phone:</span></dt>
-                <dd id="entity-business-phone">{{fullPhoneNumber}}</dd>
-              </template>
-            </dl>
-
-            <!-- if not LTD or ULC, show old menu -->
-            <v-menu v-if="!isLtdOrUlc" bottom left offset-y>
-              <template v-slot:activator="{ on }">
-                <v-btn id="entity-settings-button" small icon color="primary" v-on="on" class="mt-n4 ml-1">
-                  <v-icon small>mdi-cog</v-icon>
+            <!-- Incorporation Number -->
+            <template v-if="businessId">
+              <dt class="mr-2">Incorporation Number:</dt>
+              <dd id="entity-incorporation-number">{{ entityIncNo || 'Not Available' }}</dd>
+            </template>
+
+            <!-- NR Number -->
+            <template v-if="nrNumber">
+              <dt class="mr-2">Name Request Number:</dt>
+              <dd id="nr-number">{{ nrNumber }}</dd>
+            </template>
+
+            <!-- Email -->
+            <template v-if="businessId">
+              <dt class="mr-2">Email:</dt>
+              <dd id="entity-business-email" @click="editBusinessProfile()">
```

Comment:
Note that the click event is on the dd so that the user can click on it or any children (ie, the button itself).